frontend/awx/resources/templates/WorkflowVisualizer/Topology.tsx

```diff
@@ -92,13 +92,16 @@ export const Topology = ({
   const createVisualization = useCallback(() => {
     const newVisualization = new Visualization();
     newVisualization.setFitToScreenOnLayout(true);
-    newVisualization.registerComponentFactory(baselineComponentFactory);
+    newVisualization.registerComponentFactory(baselineComponentFactory as ComponentFactory);
```

Comment:
I spent some time trying to work through remove this typescript casting.  I was not able to, but it is something we should try to do in future iterations.

Reply:
Agreed. We can chip away at this problem in the next coming PRs.

Reply:
Fixed in follow-up commit [13033a5](https://github.com/ansible/ansible-ui/pull/1225/commits/13033a505ac09e53791ce562319a9af03ef3e854)